content/components/overview/PendingReviews.tsx

```diff
@@ -0,0 +1,49 @@
+import React from "react";
+import { Link } from "react-router-dom";
+import { Col, List, Panel, Row, Tag } from "rsuite";
+
+import { RecipeV3 } from "devtools/types/recipes";
+import { chunkBy } from "devtools/utils/helpers";
+
+export const PendingReviews: React.FC<any> = (props: {
+  data: Array<RecipeV3>;
+}) => {
+  const { data } = props;
```

Comment:
You can say this a bit more concisely (and type safely) by putting the types in the generic slot:

```suggestion
export const PendingReviews: React.FC<{ data: Array<RecipeV3> }> = ({ data }) => {
```